string cmd
string arg1
string arg2
---
string res 
